tools/travis-setup-db.sh

```diff
@@ -68,18 +71,50 @@ Debug                = 1
 ByteaAsLongVarBinary = 1
 EOL
 
-elif [ $DB = 'riak' ]; then
+elif [ "$DB" = 'riak' ]; then
     echo "Configuring Riak with SSL"
-    sudo cp ${SSLDIR}/fake_cert.pem ${RIAK_DIR}/cert.pem
-    sudo cp ${SSLDIR}/fake_key.pem ${RIAK_DIR}/key.pem
-    sudo cp ${SSLDIR}/ca/cacert.pem ${RIAK_DIR}/cacertfile.pem
-    sudo cp ${DB_CONF_DIR}/advanced.config ${RIAK_DIR}/advanced.config
-    sudo cp ${DB_CONF_DIR}/riak.conf ${RIAK_DIR}/riak.conf
-    sudo service riak restart
-    echo "Setup Riak"
-    sudo tools/setup_riak mongooseim_secret
-
-elif [ $DB = 'cassandra' ]; then
+    # Instead of docker run, use "docker create" + "docker start".
```

Comment:
What if instead of doing `create` + `start` trick we did the following?:

```bash
RIAK_CONF=$(mktemp)
docker run --rm --entrypoint bash michalwski/docker-riak:1.0.6 -c 'cat /etc/riak/riak.conf' > ${RIAK_CONF}

```

Reply:
Two times container creation in your case vs one time container creation in mine.
I want to measure both, but yours version would crash.
Because of the reason, described in a comment here in the code:

```
+    # We can't use volumes for riak.conf, because:
+    # - we want to change it
+    # - container starting code runs sed on it and gets IO error,
+    #   if it's a volume (THIS)
```

Reply:
> Two times container creation in your case vs one time container creation in mine.

I don't think this is a problem

> I want to measure both, but yours version would crash.

Yep, that's a problem